### DISCUSSION TITLE
What Is the Function of the “DBG” File?

### DISCUSSION THREAD

**Discussion by 1457384613gh** [dakanji/RefindPlus#58] [General]
What's the function of RefindPlus_x64-DBG.efi ?
How is "RefindPlus_x64-DBG.efi" different from "RefindPlus_x64-REL.efi"
？
"RefindPlus_x64-REL.efi" is to be renamed "RefindPlus_x64.efi"and put into ESP: /EFI/refindplus/ , 
but
 is "RefindPlus_x64-DBG" to be renamed "BOOTX64.EFI" and put into ESP: /EFI/BOOT/ ?

**Comment by dakanji** [comment]
There is a release version for day to day use, `RefindPlus_x64-REL.efi`, and a debug version for trouble shooting, `RefindPlus_x64-DBG.efi`.
You use one or the other (not both together) depending on what you need to do.

For instance, to raise an issue, you should attach a debug log which is produced by the debug version.

PS: Please refrain from duplicating items and from making questions as bug reports. Ask questions here and use issues for actual bug reports which must follow the issue template.

Thanks
